OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -375,6 +375,13 @@ void trackVideoOrientation(String videoId, Double currentTime,
         String FORUM_ADD_RESPONSE = "Forum: Add Thread Response";
         String FORUM_VIEW_RESPONSE_COMMENTS = "Forum: View Response Comments";
         String FORUM_ADD_RESPONSE_COMMENT = "Forum: Add Response Comment";
+        String PROFILE_VIEW = "Profile View";
+        String PROFILE_EDIT = "Profile Edit";
+        String PROFILE_CROP_PHOTO = "Crop Photo";
+        String PROFILE_CHOOSE_BIRTH_YEAR = "Choose Form Value from Birth Year";
+        String PROFILE_CHOOSE_LOCATION = "Choose Form Value from Location";
+        String PROFILE_CHOOSE_LANGUAGE = "Choose Form Value from Primary Language";
```

Comment:
Remove "form" to match up with iOS.

"Choose Form Value from Primary Language" -> "Choose Form Value Primary Language"
"Choose Form Value from Birth Year" -> "Choose Form Value Birth Year"
"Choose Form Value from Location" -> "Choose Form Value Location"